mythril/analysis/modules/integer.py

```diff
@@ -27,6 +28,19 @@
 
 
 class OverUnderflowAnnotation:
+    """ Symbol Annotation used if a BitVector can overflow"""
+
+    def __init__(
+        self, overflowing_state: GlobalState, operator: str, constraint
+    ) -> None:
+        self.overflowing_state = overflowing_state
+        self.operator = operator
+        self.constraint = constraint
+
+
+class OverUnderflowStateAnnotation(StateAnnotation):
+    """ State Annotation used if an overflow both possible and used in the annotated path"""
```

Comment:
an `is` is missing here right? "State Annotation used if an overflow is both possible and used in the annotated path"